Essentials/src/main/java/com/earth2me/essentials/PlayerTarget.java

```diff
@@ -4,15 +4,17 @@
 import org.bukkit.Location;
 import org.bukkit.entity.Player;
 
+import java.util.UUID;
+
 public class PlayerTarget implements ITarget {
-    private final String name;
+    public final UUID uuid;
```

Comment:
This is an implementation detail
```suggestion
    private final UUID uuid;
```